package.json

```diff
@@ -31,6 +31,7 @@
 		"@sapphire/discord.js-utilities": "^4.1.5",
 		"@sapphire/pieces": "^3.2.0",
 		"@sapphire/ratelimits": "^2.1.10",
+		"@sapphire/stopwatch": "^1.2.0",
```

Comment:
You'll have to run `yarn install` to update the `yarn.lock` file so your CI will pass.